TimeParse.cpp

```diff
@@ -0,0 +1,72 @@
+#include "TimeParse.h"
+#include <iostream>
+
+using namespace std;
+
+TimeParse::TimeParse(string line)
+{
+    doParsing(line);
+}
+void TimeParse::doParsing(string input)
+{
+    int set_count =0, startIndex =0;
+    string word;
+    for(int i=0;i<input.length();i++)
+    {
+        if(input[i] ==':'){
+            word = input.substr(startIndex,i-1);
+            startIndex=i+1;
+            set_count++;  
+        }
+            setMemberVariableValues(word,set_count);
```

Comment:
member variable to be set only when set_count is changed, and not to set throughout for loop

Reply:
done